auslib/test/test_db.py

```diff
@@ -1109,6 +1113,11 @@ def testEnactChangeExistingRow(self):
         self.assertEquals(history_row.data_version, 2)
         self.assertEquals(sc_row.complete, True)
 
+    def testEnactChangeForDeletingExistingRow(self):
```

Comment:
You've got a good start on the tests here, but you need to add some to verify that things work correctly when inserting, updating, and deleting scheduled changes. I'm particularly concerned that it might currently be possible to insert a scheduled change that asks for a deletion, but with no primary key defined.

You also need to update testTablesHaveCorrectColumns to check for the new column.